app/controllers/candidate_interface/find_course_selections_controller.rb

```diff
@@ -0,0 +1,54 @@
+module CandidateInterface
+  class FindCourseSelectionsController < CandidateInterfaceController
+    before_action :redirect_to_dashboard_if_not_amendable
+    rescue_from ActiveRecord::RecordNotFound, with: :render_404
+
+    def confirm_selection
+      course = Course.find(params[:course_id])
+      @course_selection_form = CourseSelectionForm.new(course)
+    end
+
+    def complete_selection
+      course = Course.find(params[:course_id])
+
+      course_selection = CourseSelectionForm.new(course, course_selection_params[:confirm])
+      if !course_selection.confirm
+        redirect_to candidate_interface_course_choices_index_path
+        return
+      end
+
+      # TODO: refactor this into a service etc.?
+      if CourseOption.where(course_id: course.id).one?
+        course_option = CourseOption.where(course_id: course.id).first
+
+        pick_site_for_course(course, course_option.id)
```

Comment:
I think we probably still need to give them the option to apply via UCAS?

Reply:
so the flow would be confirm_selection ----> apply on ucas or apply ------> interstitial page/action

Reply:
thanks @davidgisbey, that's an unintended side-effect, I'll reinstate that...

Reply:
Just had a look at this by testing manually and I still see the apply on new service or UCAS page if I hit the site with something like `/candidate/apply?providerCode=1N1&courseCode=238T` whether I am logged in or not - it's still implemented in `CandidateInterface::ApplyFromFindController`. So we are still giving the option but it's at the start of the process not after the new confirm selection page like it was before.